src/components/previewers/VideoPlayer.jsx

```
@@ -0,0 +1,180 @@
+                    {title}
+                  </div>
+                  <div className="text-gray-500 text-sm">
+                    {description?.length > 80 ? (
+                      <>
+                        {seeMore
+                          ? description
+                          : `${description.slice(0, 80)}... `}
+                        <span
+                          className={`${textColor} cursor-pointer font-medium ml-1`}
+                          onClick={() => setSeeMore(!seeMore)}
+                        >
```

Biome (1.9.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)